packages/reactivity/src/reactive.ts

```diff
@@ -405,6 +405,7 @@ export type Raw<T> = T & { [RawSymbol]?: true }
  * @see {@link https://vuejs.org/api/reactivity-advanced.html#markraw}
  */
 export function markRaw<T extends object>(value: T): Raw<T> {
+  if (hasOwn(value, ReactiveFlags.SKIP)) return value
```

Comment:
Perhaps merge this check into the `if` below, rather than duplicating the `return value`?

Reply:
Agreed